insights/parsers/satellite_version_on_client.py

```diff
@@ -0,0 +1,53 @@
+"""
+SatelliteVersionOnClient - command ``curl -k https://satellite.com:port/katello/api/status``
+============================================================================================
+
+The API returns json format of the katello version, the parser parses the
+output and return the corresponding satellite version for the satellite client.
+"""
+
+from insights.core import JSONParser
+from insights.parsers import SkipException
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+# only list supported satellite versions
+KATELLO_SATELLITE_VERSION_MAPPING = {
+    '3.14': '6.7',
+    '3.16': '6.8',
+    '3.18': '6.9'
+}
+
+
+@parser(Specs.get_satellite_version_on_client)
+class SatelliteVersionOnClient(JSONParser):
+    """
+    Parse the output of ``curl -k https://satellite.com:port/katello/api/status``.
+
+    Sample output::
+
+        {"version":"3.18.1.22","timeUTC":"2021-05-20 13:59:20 UTC"}
+
+    Attributes:
+        satellite_version (string): The satellite version which the client connects to.
```

Comment:
Since the name of the parser contains "SatelliteVersion" already, how about renaming this attribute as "version"? and add "major", "minor" just like the SatelliteVersion Combiner?
https://github.com/RedHatInsights/insights-core/blob/62dc4062b8808478aecfe737074dec7c83d50eba/insights/combiners/satellite_version.py#L106-L107

Reply:
@xiangce, I updated it, and also combine the parser in the combiner SatelliteVersion. Please help to review it again.

Reply:
@huali027 - no you should not combine this parser to the SatelliteVersion.